clients/ctl/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -131,10 +132,9 @@ export const CustomSelect = ({
         >
           <Select
             options={options}
-            onBlur={(option) => {
-              if (option) {
-                field.onBlur(props.name);
-              }
+            onBlur={(e) => {
+              setTouched({ ...touched, [field.name]: true });
```

Comment:
Why is this necessary? My guess is that formik's `onBlur` will handle this, and we just need to use [validateOnBlur](https://formik.org/docs/api/formik#validateonblur-boolean).

Reply:
`validateOnBlur` defaults to true so we are already using it, and it does work on the text input fields, but not on these slightly more bespoke `select` fields (which wrap around react-select). The select fields wouldn't render their error on blur, and when I dug into it more, it was because of [this line](https://github.com/ethyca/fides/blob/574b5a8c70f9b2b8cc46082a8c19eda057480c0a/clients/ctl/admin-ui/src/features/common/form/inputs.tsx#L117) which will only show the error if there is an error AND the field has been touched. debugging showed that the field wasn't being marked as touched, so I added this call to `setTouched` which fixed things.

not sure if there's something else going on that is preventing `setTouched` from being called automatically by formik though